src/com/phonebookservice/server/FileParser.java

```diff
@@ -0,0 +1,127 @@
+package com.phonebookservice.server;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.Writer;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+
+import com.phonebookservice.exception.InternalServerException;
+import com.phonebookservice.model.Contact;
+import com.phonebookservice.util.CollectionUtility;
+import com.phonebookservice.util.MyArrayList;
+
+public class FileParser implements ContactParser {
+    private static final String ENCODING = "UTF-8";
+    private String fileName = null;
+    private MyArrayList<Contact> myContactList = null;
+    private Map<String, Contact> myContactMap = null;
+
+    /**
+     * Initialization of file parser.
+     *
+     * @param myContactList
+     * @param myContactMap
+     */
+    public FileParser(final MyArrayList<Contact> myContactList,
```

Comment:
No it shouldn't MyArrayList, It should be Collection. 

Who is caring it is arraylist or myarraylist or tree or etc ?